dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.org.eclipse.microprofile.jwt-1.0.feature

```diff
@@ -3,6 +3,7 @@ symbolicName=com.ibm.websphere.appserver.org.eclipse.microprofile.jwt-1.0
 WLP-DisableAllFeatures-OnConflict: false
 visibility=private
 singleton=true
+-features=io.openliberty.mpCompatible-0.0
 -bundles=com.ibm.websphere.org.eclipse.microprofile.jwt.1.0; location:="dev/api/stable/,lib/";type=jar; mavenCoordinates="org.eclipse.microprofile.jwt:microprofile-jwt-auth-api:1.0"
```

Comment:
need to remove `type=jar` here, since you're now relying on this jar being a bundle

Reply:
that fixed it locally, thank you